pilot/cmd/pilot-agent/main.go

```diff
@@ -317,6 +317,10 @@ var (
 			stop := make(chan struct{})
 			cmd.WaitSignal(stop)
 			<-stop
+			requestCmd.SetArgs([]string{"POST", "/healthcheck/fail"})
```

Comment:
All new features should have flags - for agent it can be an env variable ( can be easily passed from injector )